### ISSUE TITLE
v2rayNG 1.5.16 及以后的版本不能用WhatsApp通话的问题

### ISSUE THREAD

**Issue by texabc** [2dust/v2rayNG#1067]
v2rayNG 1.5.16 及以后的版本的出现 WhatsApp打电话无法接通。
v2rayNG 1.4.13版本及以前版本没有问题。

### 预期行为
按WhatsApp电话功能，显示“响铃中”，对方接收通话后，显示“连接中”，显示“通话中”并可以开始正常通话

### 实际行为
按WhatsApp电话功能，显示“响铃中”，对方接收通话后，显示“连接中”，过后显示“通话无法接通，无法进行通话。您所使用的 Wi-Fi网络不允许WhatsApp话音通话，请关闭Wi-Fi连接或者连接到其它网络。"

**Comment by yuhan6665** [comment]
Need to know what is the **first** version that issue occur. Maybe you can try the following:
https://github.com/2dust/v2rayNG/releases/tag/1.5.2
https://github.com/2dust/v2rayNG/releases/tag/1.5.2-v2fly-core

**Comment by texabc** [comment]
Actually I tried several versions,  starting from version 1.5.16,  whatsapp call has problem,  the latest version also has same problem.  I tried the pre-release version 1.6.10,  same issue.  
Now I am using 1.4.13, this version has no issue. 

I am using a Huawei Honor 30 pro+,  UI version 4.0.0.174

**Comment by texabc** [comment]
I have just tested 1.5.2, it seems ok,  I will check again

**Comment by texabc** [comment]
today I have just tested all the versions again, but everything seems to be smooth again, I am quite confused if it was caused by my WiFi or mobile data, but I did encounter the issues with the whatsapp call, and only when I installed version 1.4.13, it became normal again.

Let me make more tests and I will update if there arises further issues.
Now I am using 1.6.10 pre-release version, which failed to make whatsapp call 2 days ago.

Thank you very much for all your efforts.
